src/redux/store.js

```diff
@@ -1,10 +1,9 @@
-import { createStore } from "redux";
+import {createStore} from "redux";
 import rootReducer from "./rootReducer";
-import initialState from "./initialState";
 import {persistStore, persistReducer} from "redux-persist";
 import LocalStorage from 'redux-persist/lib/storage'
 
-const devtoolsExtension = window.__REDUX_DEVTOOLS_EXTENSION__()
+const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
```

Comment:
read about https://redux-toolkit.js.org/api/configureStore, if you have time for that